### PR TITLE
chore: create new plugin zapp analytics plugin brightcove

### PR DESCRIPTION
**Pull request by kononenkoAnton**
## Description
Implement Brigtcove analytics 
[Monday ticket](https://applicaster.monday.com/boards/1615228456/views/35072856/pulses/3442316050?userId=16469543)

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/zapp-analytics-plugin-brightcove/apple/ZappAnalyticsPluginPlayer.podspec.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "tag": "@@applicaster/zapp-analytics-plugin-brightcove/0.3.4"
+  },
+  "requires_arc": true,
+  "public_header_files": "universal/**/*.h",
+  "source_files": "universal/**/*.{h,m,swift}",
+  "xcconfig": {
+    "CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES": "YES",
+    "ENABLE_BITCODE": "YES",
+    "SWIFT_VERSION": "5.1",
+    "OTHER_CFLAGS": "-fembed-bitcode"
```
Comment (alexzchut) [inline plugins/zapp-analytics-plugin-brightcove/apple/ZappAnalyticsPluginPlayer.podspec.json:27]: remove line 27 and line 25

File: plugins/zapp-analytics-plugin-brightcove/manifests/manifest.config.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
+const min_zapp_sdk = {
+  ios_for_quickbrick: "6.0.0-Dev",
```
Comment (alexzchut) [inline plugins/zapp-analytics-plugin-brightcove/manifests/manifest.config.js:2]: 6.3.0-Dev

File: plugins/zapp-analytics-plugin-brightcove/manifests/manifest.config.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  android_for_quickbrick: "8.1.0",
+  android_tv_for_quickbrick: "8.1.0",
+  amazon_fire_tv_for_quickbrick: "8.1.0",
+  tvos_for_quickbrick: "6.0.0-Dev",
```
Comment (alexzchut) [inline plugins/zapp-analytics-plugin-brightcove/manifests/manifest.config.js:6]: 6.3.0-Dev

**Review by alexzchut — CHANGES_REQUESTED** [review]

**Comment by inckie** [comment]
On Android I see wrong deviceType `deviceType=other`
Also I see duration reported with dot, hope backend will work with that `video_duration=154529.0`

**Review by inckie — APPROVED** [review]

**Pull request merged**